edit/edit.php

```diff
@@ -0,0 +1,28 @@
+<?php
+$db = new PDO('mysql:host=192.168.20.20;dbname=cms_portfolio', 'root', '');
+$db-> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$getProjects = 'SELECT `id`, `project_title`, `img_url`, `site_url` FROM projects WHERE `id` = ?';
+
+
+$query = $db->prepare($getProjects);
+$query->execute([$_GET['id']]);
+
+$project = $query->fetch();
+
+
+echo '<html lang="en"></html><link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
```

Comment:
if you close your php tag you can put all this html as just text instead of in an echo